### DISCUSSION TITLE
Session example please

### DISCUSSION THREAD

**Discussion by etc-austin** [adonisjs/core#3345] [Help] [answered]
Playing with sessions just to see how to use them and I can't get what I think is something very easy to work. The error is get is not a function

```
import { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'
import session from "Config/session";

export default class CompanyContactsController {
  public async index ({ view }: HttpContextContract){
    const AC = session.get('area_code')

    // Write value
    session.put('area_code', 777)
    return view.render('company_contact')
  }

}
```
I have config/session.ts installed and configured according to the docs.

What am I missing to define a session variable?

**Comment by thetutlage** [comment, marked as answer]
Where did you find this example in the docs?

**Reply by etc-austin** [reply]
Never mind I wasnt passing the session to the function. I assumed it worked like a global
